<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorColorize</class>
 <widget class="QDialog" name="ColorColorize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单色化</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Image.qrc">
    <normaloff>:/icon/color/Image/stock-color-colorize.png</normaloff>:/icon/color/Image/stock-color-colorize.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>选择颜色</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>色调：  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_H">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_H">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>180</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>饱和度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_S">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_S">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>亮度：  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_L">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_L">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider_H</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_H</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_H</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_H</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_S</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_S</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_S</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_S</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_L</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_L</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_L</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_L</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>ColorColorize</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ColorColorize</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
